cli/tests/test_git_push.rs

```diff
@@ -279,6 +279,25 @@ fn test_git_push_changes() {
     Branch changes to push to origin:
       Force branch push-yostqsxwqrlt from 48d8c7948133 to b5f030322b1d
     "###);
+    // Test changing `git.push-branch-prefix`. It causes us to push again.
+    let stdout = test_env.jj_cmd_success(
+        &workspace_root,
+        &[
+            "git",
+            "push",
+            "--config-toml",
+            r"git.push-branch-prefix='test-'",
+            "--change",
+            "@",
+            "--change",
+            "@",
```

Comment:
The repeated `--change @` isn't relevant at this point any more, right?

Reply:
I thought it tests slightly more this way, but you're right. I can remove it if you prefer.

Reply:
I agree that it's clearer without it

Also, you can now use `-c` instead of `--change`, and even `-c=@` if you like (maybe that reduces line wrapping?)

Reply:
Sounds good, thanks Glen and Martin!